service/src/main/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiController.java

```diff
@@ -70,8 +73,8 @@ public ResponseEntity<ApiCreatedControlledGcpGcsBucket> createBucket(
         toCommonFields(workspaceUuid, body.getCommon(), userRequest);
     ControlledGcsBucketResource resource =
         ControlledGcsBucketResource.builder()
-            .bucketName(body.getGcsBucket().getName())
```

Comment:
PS: swap bucketname after the common because get bucket name require common. (also for other swaps in other files.)
in service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/gcsbucket/ControlledGcsBucketResource.java. `bucketName = ControlledGcsBucketHandler.getHandler().generateCloudName(common.getWorkspaceId(), common.getName());`